### ISSUE TITLE
A11y_Azure Extensions for VS Code_Default Screen_screenreader: screenreader is not announcing expand or collapse

### ISSUE THREAD

**Issue by Yash14j** [microsoft/vscode-azurefunctions#2227]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHub Tags:#A11yTCS;#Azure Tools for VS Code;#A11y_Azure Extensions for VS Code; #DesktopApp;#Win10;#MAS1.3.1;#Benchmark;#FTP;#STP;#NVDA;#JAWS;#A11yMAS;;#A11ySev2;

Environment Details:
Application Name: Azure Extensions for VS Code
Version: 1.46.1 (user setup)
Commit: cd9ea6488829f560dc949a8b2fb789f3cdc05f5d
Date: 2020-06-17T21:13:20.174Z
Electron: 7.3.1
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Windows Version: Windows10

**Repro Steps:**

1. Open Visual studio code 
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button
4. Default screen will get open.
5. Open screenreader and expand local project or function with space key or arrow key and check the result

**Actual:**

When we try to expand or collapse the local project or function with the help of space key or arrow key screenreader does not announces expanded or collapse.

**Same issue is also observed with throughout the application.
Same issue is also observed with both NVDA and JAWS.**

**Expected:**

Screenreader should announce expand or collapse when done.

**User Impact**:

User will get confused and will not get proper information weather it has expanded/collapsed or not

**Attachment:**
[#2227.zip](https://github.com/microsoft/vscode-azurefunctions/files/4889405/2227.zip)

**Comment by EricJizbaMSFT** [comment]
This was fixed in VS Code 1.47. https://github.com/microsoft/vscode/issues/99563

I see "expanded" and "collapsed"
<img width="728" alt="Screen Shot 2020-07-20 at 2 21 03 PM (3)" src="https://user-images.githubusercontent.com/11282622/87987992-6663ab80-ca94-11ea-8664-7e0219f6da2f.png">

**Comment by harshgangrade111** [comment]
@EricJizbaMSFT This issue should not be close by product team please assign this bug to tester for verification will verify the issue than will close it

**Comment by Yash14j** [comment]
This issue does not repro in the Visual studio code version 1.49.0, extension version 0.24.0 hence closing the bug. Please refer the attachment
[2227_closed.zip](https://github.com/microsoft/vscode-azurefunctions/files/5223626/2227_closed.zip)
